test/swift/EngineBuilderTests.swift

```diff
@@ -586,6 +594,13 @@ final class EngineBuilderTests: XCTestCase {
     XCTAssertTrue(resolvedYAML.contains("&dns_multiple_addresses false"))
     XCTAssertTrue(resolvedYAML.contains("&enable_interface_binding false"))
     XCTAssertTrue(resolvedYAML.contains("&trust_chain_verification VERIFY_TRUST_CHAIN"))
+    XCTAssertTrue(resolvedYAML.contains(
+"""
+&validation_context
+  custom_validator_config:
+    name: \"envoy_mobile.cert_validator.platform_bridge_cert_validator
```

Comment:
Is this the expected string? Has an over-escaped leading quote and no trailing quote? There's no need to escape quotes like this in a multiline string literal.

Reply:
Hm, yeah that's confusing. I had some trouble sorting out multi-line strings in Swift and I think it lost the trailing quote. Fixed! Thanks,